raiden-dapp/tests/unit/raiden-service.spec.ts

```diff
@@ -659,4 +662,16 @@ describe('RaidenService', () => {
       expect(transfer).toHaveBeenCalledWith('0xtoken', '0xpartner', One);
     });
   });
+
+  it('resolve ens addresses', async () => {
```

Comment:
Added this test to increase overall code coverage because codecov complains that https://github.com/raiden-network/light-client/pull/308/files#diff-ddfc85a29ab30ce1822e4b860e4305d2R268 isn't hit during tests and I didn't manage to get it hit.

Reply:
Ok, this is not ideal, but I don't see big deal with it right now, this part of the code will need to be reworked when implementing pending/parallel transfers badge/dropdown, so then we can add proper coverage on this